src/views/Token/index.tsx

```diff
@@ -0,0 +1,151 @@
+// External
+import React, { useState, useEffect } from 'react'
+import dayjs from 'dayjs'
+import { ethers } from 'ethers'
+import { BigNumber } from 'ethers'
+import { useTranslation } from 'react-i18next'
+import { useDispatch } from 'react-redux'
+
+// Redux
+import { setPageTitle } from 'src/redux/page'
+
+// Components
+import { AbsoluteContainer } from 'src/components/AbsoluteContainer'
+import { Container } from 'src/components/Container'
+import { Header } from 'src/components/Header'
+import { Footer } from 'src/components/Footer'
+import { Title } from 'src/components/Title'
+
+// Hooks
+import { useWindowSize } from 'src/hooks/useWindowSize'
+import { useMesa } from 'src/hooks/useMesa'
+
+// Layouts
+import { GridListSection } from 'src/components/Grid'
+
+// interface
+import { FixedPriceSalePurchase } from 'src/interfaces/Sale'
+import { ErrorMesssage } from 'src/components/ErrorMessage'
+
+// claims
+import { TokenClaim } from './components/TokenClaim'
+
+const userSalesQuery = (userAddress: string) => `
+
+query getTokenClaims {
+  fixedPriceSalePurchases(where:{
+    buyer:"${userAddress}"})
+  {
+    id
+    amount
+    sale {
+        id
+      soldAmount
+      minimumRaise
+      name
+      status
+      startDate
+      endDate
+      tokenPrice
+      sellAmount
+      tokenIn {
+        id
+        name
+        symbol
+        decimals
+      }
+      tokenOut {
+          id
+        name
+        symbol
+        decimals
+      }
+      allocationMin
+      allocationMax
+
+    }
+  }
+}
```

Comment:
It would be a good practice (and a new skill to pick up) for you to learn how to generate queries using the Apollo CLI. For this PR, leave this as it is because it will create merge conflict again the base branch.